src/editors/containers/TextEditor/hooks.js

```diff
@@ -110,24 +109,20 @@ export const editorConfig = ({
   },
   initialValue: blockValue ? blockValue.data.data : '',
   init: {
-    ...pluginConfig(isLibrary).config,
-    skin: false,
-    content_css: false,
-    content_style: tinyMCEStyles,
+    ...tinyMCEConfig(isLibrary).config.shared,
+    ...tinyMCEConfig(isLibrary).config.textEditor,
     contextmenu: 'link table',
     document_base_url: lmsEndpointUrl,
     init_instance_callback: module.checkRelativeUrl(module.fetchImageUrls(images)),
     imagetools_cors_hosts: [removeProtocolFromUrl(lmsEndpointUrl), removeProtocolFromUrl(studioEndpointUrl)],
```

Comment:
Should we add these to the problem editor config? As it will eventually need the images tool, or is that for Kristin's upcoming work you think?

Reply:
That is part of my work